lib/app/features/posts/view/favorites_view.dart

```diff
@@ -9,16 +10,16 @@ class FavoritesView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) => Scaffold(
-        appBar: AppBar(title: const Text('Favorites')),
+        appBar: AppBar(title: const Text('Favorites'), actions: [SettingsIconButton()]),
         body: Center(
           child: Consumer(
             builder: (context, ref, child) {
               final posts = ref.watch(favoriteListProvider);
-              debugPrint(posts.valueOrNull?.map((e) => '${e.id}').toString());
+              debugPrint("[Favorites Stream] ${posts.valueOrNull?.map((e) => '${e.id}')}");
```

Comment:
Please make a ticket to add logger or talker or both

Reply:
Linked here: [WA-35](https://anirac-tech.atlassian.net/browse/WA-35?atlOrigin=eyJpIjoiODBkMWZjOTI5M2JmNDg5NmJiMDY2Njg2MTJmM2IwNzAiLCJwIjoiaiJ9)